### PR TITLE
Add help message in doc:go-client-api.md

### PR DESCRIPTION
**Pull request by hhhowyou**
## What have you changed? (mandatory)

I met this build failed error in my usage, so I add it to the doc. Maybe it will help others.

Add help message in doc/client/go-client-api.md.

### CODE REVIEW

File: docs/clients/go-client-api.md [hunk 1]
```diff
@@ -124,6 +124,16 @@ found val:  for key: Company
 
 RawKVClient is a client of the TiKV server and only supports the GET/PUT/DELETE/SCAN commands. The RawKVClient can be safely and concurrently accessed by multiple goroutines, as long as it is not closed. Therefore, for one process, one client is enough generally.
 
+### Possible Error
```
Comment (siddontang) [inline docs/clients/go-client-api.md]: @disksing 

I think we should update our doc to use client-go directly, not use tidb.

**Comment by hhhowyou** [comment]
PTAL @ice1000

File: docs/clients/go-client-api.md [hunk 2]
```diff
@@ -124,6 +124,16 @@ found val:  for key: Company
...
+
+If you get the error like follows.
```
Comment (ice1000) [inline docs/clients/go-client-api.md]:
```suggestion
If you see this error:
```

**Comment by ice1000** [comment]
Thanks for your contribution!

**Pull request merged**